atlasdb-ete-tests/docker/conf/atlasdb-ete.dbkvs.yml

```diff
@@ -21,6 +21,8 @@ atlasdb:
       dbName: atlas
       dbLogin: palantir
       dbPassword: palantir
+      connectionParameters:
+        sslmode: disable
```

Comment:
isn't this going to get overwritten? looks like you want a clause in the config overwriting if sslmode is explicitly set to disable.

Reply:
If it was overwritten the tests wouldn't work ;)

We apply the default `sslmode` before applying custom parameters. This ensure that consumers can still overwrite this if they want.

Reply:
ugh I can't read, shouldn't be reviewing code at this time

of course, you're overriding it in the map